scripts/post-update.sh

```diff
@@ -235,6 +235,44 @@ if ! git remote | grep -q local; then
     git remote add local ~/companion
 fi
 
+#Install isc-dhcp-server
+cd $HOME/companion
+echo "Checking for isc-dhcp-server..."
+while true
+do
+    if dpkg-query -W -f='${Status} ${Version}' isc-dhcp-server | grep 4.3; then
+        echo "Already installed"
+        sudo dpkg --configure -a
+        break
+    else
+        echo "Installing isc-dhcp-server"
+        sudo apt-get update --yes
+        sudo apt-get install isc-dhcp-server=4.3.* --yes
+        sudo dpkg --configure -a
+        echo "isc-dhcp-server installed"
+    fi
+done
+
+#Delete ip address if present from /boot/cmdline.txt
+# e.g. sed command removes any ip address with any combination of digits [0-9] between decimal points
+sudo sed -i -e 's/\s*ip=[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*//' /boot/cmdline.txt
+
+#Copy network configuration files from Companion directory to respective configuration directories
+sudo cp /home/pi/companion/eth0-config /etc/network/interfaces.d/
```

Comment:
why do we need sudo to copy this file?

Reply:
because it wasnt able to copy in the interfaces.d directory without root permissions. but i guess we are running post-update as root anyways. let me make sure it works without sudo and i will change it. because i remember testing it once and it wasnt working.

Reply:
my mistake I did not see the /etc here.